sdk/cosmos/azure-cosmos/test/test_crud_async.py

```diff
@@ -90,28 +93,40 @@ async def __AssertHTTPFailureWithStatus(self, status_code, func, *args, **kwargs
         """
         try:
             await func(*args, **kwargs)
-            self.assertFalse(True, 'function should fail.')
+            pytest.fail('function should fail.')
         except exceptions.CosmosHttpResponseError as inst:
-            self.assertEqual(inst.status_code, status_code)
+            assert inst.status_code == status_code
 
     @classmethod
-    async def setUpClass(cls):
-        cls.client = CosmosClient(cls.host, cls.masterKey, consistency_level="Session", connection_policy=cls.connectionPolicy)
-        cls.databaseForTest = await cls.client.create_database_if_not_exists(test_config._test_config.TEST_DATABASE_ID)
-
-    async def test_database_crud(self):
+    async def _set_up(cls):
+        if (cls.masterKey == '[YOUR_KEY_HERE]' or
+                cls.host == '[YOUR_ENDPOINT_HERE]'):
+            raise Exception(
+                "You must specify your Azure Cosmos account values for "
+                "'masterKey' and 'host' at the top of this class to run the "
+                "tests.")
+
+        cls.client = CosmosClient(cls.host, cls.masterKey)
+        cls.database_for_test = await cls.client.create_database_if_not_exists(
+            test_config._test_config.TEST_DATABASE_ID)
+
+    async def _clear(self):
```

Comment:
Good addition, we should make sure to clear up resources, are we performing this for every test class? If not, worth adding.

Reply:
we already do a cleaning of the test account after we're done with the tests, but I added this method in this case since it seems some resources were being recycled by the tests in the same file and we needed to refresh those.